values.yaml

```diff
@@ -409,9 +409,6 @@ controllerteam:
   args:
   - "controller"
   - "team"
-  image:
```

Comment:
hopefully use what it in the [`requirements.yaml`](https://github.com/jenkins-x/jenkins-x-platform/blob/89a4725dd4967c354b7cbc0328941c4a1260533d/requirements.yaml#L56-L60)